src/Test/PdbUtilities/Shared/SymUnmanagedReaderExtensions.cs

```diff
@@ -332,7 +73,7 @@ internal static ImmutableArray<string> GetLocalVariableSlots(this ISymUnmanagedM
             }
 
             // recurse:
-            var children = GetScopesInternal(scope);
+            var children = scope.GetChildren();
             if (children != null)
```

Comment:
This can be further simplified. GetChildren doesn't return null anymore, but an empty array.